tic_tac_toe/game.py

```diff
@@ -24,16 +26,26 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if (not isinstance(position, tuple) 
+        or not (isinstance(position[0], int) and isinstance(position[1], int))
+        or not len(position) == 2
+        or not ((0 <= position[0] < 3) and (0 <= position[1] < 3))):
+            
```

Comment:
Line 32 actually just makes sure what value is input lands between 0 and 3 (as per your suggestion, we should definitely adjust the check there). But without confirming the values are actually ints(line 30), a user could enter 1.2 or something and not return False.